hook_event_dispatcher.module

```diff
@@ -354,6 +355,19 @@ function hook_event_dispatcher_entity_base_field_info(EntityTypeInterface $entit
   return $event->getFields();
 }
 
+/**
+ * Implements hook_entity_base_field_info_alter().
+ *
+ * {@inheritdoc}
+ */
+function hook_event_dispatcher_entity_base_field_info_alter(array &$fields, EntityTypeInterface $entityType) {
+  /* @var \Drupal\hook_event_dispatcher\Manager\HookEventDispatcherManagerInterface $manager */
+  $manager = \Drupal::service('hook_event_dispatcher.manager');
+  $event = new EntityBaseFieldInfoAlterEvent($fields, $entityType);
+  $manager->register($event);
+  return $event->getFields();
```

Comment:
Ah sorry mist this in the first round. My last nitpick, please remove this line as the by reference fields is used and not the returned array.